src/main/docs/guide/config-client.adoc

```diff
@@ -60,6 +60,20 @@ kubernetes:
         - env: prod
 ----
 
+Or by including every config map that has the same Kubernetes label as the pod.
```

Comment:
I would  extend the documentation to be clear about what pad ..  `...Kubernetes label as the pod the micronaut application is running in.`

Reply:
done